docs/user/unofficial-projects.rst

```diff
@@ -0,0 +1,134 @@
+Policy for Unofficial and Unmaintained Projects
+===============================================
+
+This policy describes a process where we take actions against unmaintained_ and unofficial_ forks of project documentation.
+
+
+Rationale
+---------
+
+Documentation projects may be kept online indefinitely, even though a newer version of the same project exists elsewhere.
+There are many reasons this can happen,
+including forks, old official docs that are unmaintained, and many other situations.
+
+The problem with old, outdated docs is that users will find them in search results,
+and get confused to the validity of them.
+Projects will then get support requests from people who are using an old and incorrect documentation version.
+
+We have this policy to allow project maintainers to request the *delisting* of forks that are old and outdated.
+
+
+Examples
+~~~~~~~~
+
+Common cases covered by this policy are:
+
+Outdated docs
+    Documentation that is not being actively maintained or updated. This means for instance that the source project has updates that are not reflected in the documentation or a newer official documentation exists elsewhere on the internet.
+
+Unofficial forks
+    Forks of a project documentation that are not linked to from the project in any way.
+
+
+Specification
+-------------
+
+
+Unofficial projects
+~~~~~~~~~~~~~~~~~~~
+
+A project is considered *unofficial* when it is not linked to or mentioned in *any* of the following places:
+
+* Websites and domains associated with the project
+* Social media accounts associated with the project
```

Comment:
This feels hard to define or check. Are we really going to figure out a projects social media?

Reply:
If it means scrolling through their entire history of tweets and instas.... NO! :)

Reply:
Removed SOME from the list.